extension/export_HartreeFockProvider.cc

```diff
@@ -303,10 +303,10 @@ class PyHartreeFockProvider : public HartreeFockProvider {
   void fill_fock_ff(py::tuple slices, py::array out) const override {
     PYBIND11_OVERLOAD_PURE(void, HartreeFockProvider, fill_fock_ff, slices, out);
   }
-  void fill_eri_ffff(py::tuple slices, py::array out) const {
+  void fill_eri_ffff(py::tuple slices, py::array out) const override {
```

Comment:
Nice catch!

Reply:
Not my catch though... Caught by Apple Clang 😄

Reply:
:+1: